hack/make-rules/verify.sh

```diff
@@ -30,9 +30,24 @@ EXCLUDED_PATTERNS=(
   "verify-linkcheck.sh"          # runs in separate Jenkins job once per day due to high network usage
   "verify-test-owners.sh"        # TODO(rmmh): figure out how to avoid endless conflicts
   "verify-*-dockerized.sh"       # Don't run any scripts that intended to be run dockerized
-  "verify-typecheck.sh"          # runs in separate typecheck job
   )
 
+# Exclude typecheck in certain cases, if they're running in a separate job.
+if [[ ${EXCLUDE_TYPECHECK:-} =~ ^[yY]$ ]]; then
```

Comment:
to echo what I said in person: I also wonder if it'd be better to pass a space-separated list of patterns to skip, rather than adding envvars for each type of check here

Reply:
to follow up on in-person conversation: it probably makes sense to add environment variables for these group of things, since we might factor out codegen checks next.